app/components/Project/ProjectFundingStreamForm.tsx

```diff
@@ -0,0 +1,65 @@
+import type { JSONSchema7 } from "json-schema";
+import React, { useMemo } from "react";
+import FormBase from "../Form/FormBase";
+import FormComponentProps from "../Form/FormComponentProps";
+import { ProjectFundingStreamForm_query } from "__generated__/ProjectFundingStreamForm_query.graphql";
+import { graphql } from "react-relay/hooks";
+import { createFragmentContainer } from "react-relay";
+
+interface Props extends FormComponentProps {
+  query: ProjectFundingStreamForm_query;
+}
+
+const uiSchema = {
+  title: "Funding Stream",
```

Comment:
should this property be `ui:title`? Otherwise this implies that the form data has a `title` attribute